src/components/Blinky.js

```diff
@@ -17,7 +18,14 @@ export default function Blinky({
   b = kInterstitialCharacter,
 }) {
   const [value, setValue] = useState(true);
-  useInterval(() => setValue(val => !val), 1000);
+  const now = useRef(new Date());
+
+  useInterval(() => {
+    // only start blinking if we've elapsed enough time
+    if (new Date() - now.current > kBlinkAfterMs) {
+      setValue(val => !val);
+    }
+  }, 1000);
```

Comment:
Hmm, doesn't this only fire once a second, reducing the precision of `kBlinkAfterMs`?

Reply:
yup! figured precise timing wasn't a big deal here. with a blink after of `2500`, it should fire on the 3rd second.